src/SettlementController.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.22;
+
+import {CreditLine} from "./CreditLine.sol";
+import {JaneBurner} from "./jane/JaneBurner.sol";
+import {JaneToken} from "./jane/JaneToken.sol";
+import {MarketParams} from "./interfaces/IMorpho.sol";
+
+/// @title SettlementController
+/// @author 3Jane
+/// @notice Atomically burns a borrower's JANE tokens and settles their account
+/// @dev Designed to be set as the OZD address in CreditLine to enable atomic settlement with token burns
+contract SettlementController {
```

Comment:
thought about it more - what if we transfer this logic over to the markdown. I think it makes sense that it is a linear burn. We whitelist the address we want to markdown anyway.

Reply:
done in another branch